### PR TITLE
Add in functionality to see all reminders

### PR DESCRIPTION
**Pull request by zkc**
## Purpose

Address Issue 1 where reminders needed to be displayed.

## Approach

Iterates over a displays reminders

### Learning

Struggled with basic ember for a bit, then looked at classmates code for reference. 

### Open Questions and Pre-Merge TODOs

My faker.lorem.words() call in the Factory returns a string. This gave me a Type error when the original code's .join() was attempted to be called. Doesn't seem like a big deal but wanted to mention it. 

### Test coverage 

Ember auto tests only. Commented out failing test to save for later.

### Follow-up tasks

- Issue #2

### CODE REVIEW

File: app/routes/reminders.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import Ember from 'ember';
+
+export default Ember.Route.extend({
+  model() {
+    return this.get('store').findAll('reminder')
```
Comment (Tman22) [inline app/routes/reminders.js]: Be consistent with semi-colons. See file above ^

**Comment by zkc** [comment]
@Tman22 @martensonbj I did it!

File: tests/unit/models/reminder-test.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import { moduleForModel, test } from 'ember-qunit';
+
+moduleForModel('reminder', 'Unit | Model | reminder', {
+  // Specify the other units that are required for this test.
+  needs: []
+});
```
Comment (Tman22) [inline tests/unit/models/reminder-test.js]: Take this test out of you're not going to use it. I'm only expecting you to write acceptance tests

File: tests/unit/routes/reminders-test.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
+import { moduleFor, test } from 'ember-qunit';
+
+moduleFor('route:reminders', 'Unit | Route | reminders', {
+  // Specify the other units that are required for this test.
+  // needs: ['controller:foo']
```
Comment (Tman22) [inline tests/unit/routes/reminders-test.js]: Also take this test out. One of these boilers will be created for you every time you use `ember g route etc`

**Comment by zkc** [comment]
woah look at that, github comments notices the changes. @Tman22 @martensonbj corrections are pushed up.

**Comment by martensonbj** [comment]
👍

**Pull request merged**